packages/auto-design-system/src/components/common/FilePreview/FilePreview.stories.tsx

```diff
@@ -1,26 +1,17 @@
-import { OffchainMetadata } from '@autonomys/auto-dag-data'
-import { NetworkId } from '@autonomys/auto-utils'
-import type { Meta, StoryObj } from '@storybook/react'
-import React, { useCallback, useEffect, useState } from 'react'
-import {
-  fileMock1,
-  fileMock2,
-  // fileMock3,
-  // fileMock4,
-  // fileMock5,
-  // fileMock6,
-  // fileMock7,
-  fileMock8,
-} from '../../../mocks/files.mock'
-import type { FileData } from '../../../types'
 import {
   canDisplayDirectly,
   decryptFileData,
-  fileGatewayUrl,
   needsContentParsing,
+  OffchainMetadata,
   processFileData,
-} from '../../../utils'
+} from '@autonomys/auto-dag-data'
+import { NetworkId } from '@autonomys/auto-utils'
+import type { Meta, StoryObj } from '@storybook/react'
+import React, { useCallback, useEffect, useState } from 'react'
+import { fileMock1, fileMock2 } from '../../../mocks/files.mock'
+import type { FileData } from '../../../types'
```

Comment:
Should this be imported from `auto-dag-data` and removed from `auto-design-system` types? See bugbot [comment](https://github.com/autonomys/auto-sdk/pull/443#pullrequestreview-3001744677).